### DISCUSSION TITLE
How can I rejoin zed channel again

### DISCUSSION THREAD

**Discussion by wingyplus** [zed-industries/zed#14708] [Feature Requests]
I just accident leave `zed` channel and I cannot see the way to find a public channel from the editor. How can I rejoin then channel again?

**Comment by SomeoneToIgnore** [comment]
![image](https://github.com/user-attachments/assets/c01c3583-fae8-43c6-9ad6-4852c9f0ef45)

There's a collab panel with a filter input in the bottom, could that help?

**Reply by wingyplus** [reply]
<img width="123" alt="Screenshot 2567-07-18 at 13 54 40" src="https://github.com/user-attachments/assets/cc2f00d1-6d55-404a-810f-d24612f8f46d">

I didn't realize that it has filter input at the bottom! But still didn't see the channel. :(

**Reply by SomeoneToIgnore** [reply]
Oh, I see, left entirely.
I think, using a channel link might help (https://zed.dev/channel/zed-283) but I've also went on and re-invited you back this time.

**Reply by wingyplus** [reply]
Thank you. ❤️
